### ISSUE TITLE
Tuning RBF

### ISSUE THREAD

**Issue by ElliottP-13** [cullingsj/Machine_Learning#24]
We need a way to tune the learning rate and sigma.

Sigma is easy for clustering as we can calculate the variance of the clusters. However, without clusters we need a way to pick them.
Learning rate must also be arbitrarily picked.

We can more intelligently tune these parameters by doing a space search. Basically we pick different values for each and move them to find the 'optimal' point. This can be very very expensive and need a way to narrow this down. We also don't need to do this method, but it would be nice to have some sort of automated tuning.

- [ ] Learning rate 
- [ ] Sigma

**Comment by cullingsj** [comment]
KMetroids and CKNN are the same for this as long as CKNN is used as metroids

**Comment by cullingsj** [comment]
The above comment refers to sigma

**Comment by cullingsj** [comment]
- Learning Rate: Grid/Line search, use the output from this to give a starting point for gradient descent.
